### PR TITLE
Updated require versions ready for 1.0.0 release

### PR DESCRIPTION
**Pull request by asgrim**
For #159 and 1.0.0 release in general

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,15 +4,16 @@
...
-        "phpdocumentor/type-resolver": "^0.1.5",
+        "phpdocumentor/type-resolver": "^0.1.6",
         "zendframework/zend-code": "^3.0",
         "jeremeamia/superclosure": "^2.2"
     },
     "require-dev": {
         "fabpot/php-cs-fixer": "^1.8",
-        "phpunit/phpunit": "^4.6"
+        "phpunit/phpunit": "^4.6",
+        "phly/changelog-generator": "^2.1"
```
Comment (Ocramius) [inline composer.json]: This isn't needed for normal dev environments. You should install it separately on your machine

File: composer.json [hunk 2]
```diff
@@ -4,15 +4,16 @@
     "license": "MIT",
     "require": {
         "php": ">= 5.6",
```
Comment (Ocramius) [inline composer.json]: We should use constrained upper bounds: `^5.6 || ^7.0`

**Comment by asgrim** [comment]
@Ocramius amended and pushed

**Comment by asgrim** [comment]
Checks passed, all fixed :+1:

**Comment by Ocramius** [comment]
:shipit:

**Pull request merged**